Shwetang550.md

```diff
@@ -49,10 +49,7 @@
 
 <p align="center"> 
   Visitor count<br>
-  <img src="https://profile-counter.glitch.me/Shwetang550/count.svg" />
+  <img src="https://avatars.githubusercontent.com/u/67409054?s=60&v=4" />
 </p>
 
 -----
-Credits: [Shwetang550](https://github.com/Shwetang550)
```

Comment:
Please keep Credits line as it is.
if you modify the file then just update last edited date.
thanks.

Reply:
Ok thanks